flink-runtime/src/test/java/org/apache/flink/runtime/security/token/KerberosDelegationTokenManagerTest.java

```diff
@@ -79,4 +93,48 @@ public void testAllProvidersLoaded() {
         assertTrue(ExceptionThrowingDelegationTokenProvider.constructed);
         assertFalse(delegationTokenManager.isProviderLoaded("throw"));
     }
+
+    @Test
+    public void isRenewalPossibleMustGiveBackFalseByDefault() throws IOException {
+        UserGroupInformation ugi = PowerMockito.mock(UserGroupInformation.class);
+        PowerMockito.mockStatic(UserGroupInformation.class);
+        when(UserGroupInformation.getCurrentUser()).thenReturn(ugi);
+
+        ExceptionThrowingDelegationTokenProvider.enabled = false;
+        Configuration configuration = new Configuration();
+        KerberosDelegationTokenManager delegationTokenManager =
+                new KerberosDelegationTokenManager(configuration);
+
+        assertFalse(delegationTokenManager.isRenewalPossible());
+    }
+
+    @Test
+    public void isRenewalPossibleMustGiveBackTrueWhenKeytab() throws IOException {
```

Comment:
We should use the junit5 resources for temporary files. Eg.

```suggestion
    public void isRenewalPossibleMustGiveBackTrueWhenKeytab(@TempDir Path tmpDir) throws IOException {
        final Path file = Files.createFile(tmpDir.resolve("test.keytab"));
```

Reply:
Fixed.